Squest/ldap_config.py

```diff
@@ -6,18 +6,20 @@
 # -----------------------
 # LDAP auth backend
 # -----------------------
-AUTH_LDAP_SERVER_URI = "ldap:port"
-AUTH_LDAP_BIND_DN = "cn=service_account_name,ou=Applications,o=domain.com"
+AUTH_LDAP_SERVER_URI = os.environ.get('AUTH_LDAP_SERVER_URI', "ldap:port")
+AUTH_LDAP_BIND_DN = os.environ.get('AUTH_LDAP_BIND_DN', "cn=service_account_name,ou=Applications,o=domain.com")
 AUTH_LDAP_BIND_PASSWORD = os.environ.get('AUTH_LDAP_BIND_PASSWORD', None)
-AUTH_LDAP_USER_SEARCH = LDAPSearch("ou=People,o=domain.com", ldap.SCOPE_SUBTREE, "(uid=%(user)s)")
-LDAP_CA_FILE_PATH = "/usr/local/share/ca-certificates/ldap_ca.crt"
+AUTH_LDAP_USER_SEARCH = LDAPSearch(os.environ.get('AUTH_LDAP_SEARCH', "ou=People,o=domain.com"), ldap.SCOPE_SUBTREE, "(uid=%(user)s)")
+LDAP_CA_FILE_PATH = os.environ.get('LDAP_CA_FILE_PATH', "/usr/local/share/ca-certificates/ldap_ca.crt")
+
 AUTH_LDAP_CONNECTION_OPTIONS = {
     ldap.OPT_X_TLS_CACERTFILE: LDAP_CA_FILE_PATH,
-    ldap.OPT_X_TLS_REQUIRE_CERT: ldap.OPT_X_TLS_NEVER,
+    ldap.OPT_X_TLS_REQUIRE_CERT: ldap.OPT_X_TLS_ALLOW,
     ldap.OPT_X_TLS_NEWCTX: 0
 }
+
 AUTH_LDAP_USER_ATTR_MAP = {
```

Comment:
Fields here should be wrapped as well then to be consistent. Like in [Netbox project](https://github.com/netbox-community/netbox-docker/blob/release/configuration/ldap/ldap_config.py#L99).

Reply:
Same comment for all other fields actually. Like TLS config.

Reply:
leaved the AUTH_LDAP_CONNECTION_OPTIONS as is but wrapped all others feilds, and add a bit of documentation.